src/sass/_recommendations_page.scss

```diff
@@ -1,5 +1,11 @@
 $card-gap: 24px;
 
+.recommendations-container {
```

Comment:
is this working for all screen types?

Reply:
The large and medium screens were already working fine and still working fine. The issue was with only a small screen and now it's working fine for all screen sizes.

Reply:
okay, so I will explain my concern in detail. The 100vh means 100% of the viewport height. right? So that means if we apply 100% of the viewport height to a sub-div of the page (which is in our case just the recommendations container) will increase the height of that div to 100vh thus creating useless scroll behavior. 

The 100vh class should be applied to the main div of the page, which contains all the content of the page. which is the `<div className="d-flex flex-column bg-light-200">` div in recommendations page. This will give 100% viewport height to the complete page instead of just a sub-div of the main page. 

But then I anticipate one more problem, our page height is not always 100vh, We have a 3 recommendations list that increases the height of the page from 100vh, so if we apply 100vh, the background color will be applied to only 100vh height part of page. So in that case we should apply a minimum height of 100vh. That will solve the issue everywhere.

we just need to put the `min-vh-100` class in the main div of the page (mentioned above).

IMO this will solve the issue. 

The reason why it is not an issue on larger screens is that the page height on larger screens is greater than 100vh so don't notice the issue there but if we have only one recommendations list, we will start noticing the issue there as well. So applying `min-vh-100` will resolve all these issues.

Please let me know if this creates any further issue.